numpy/core/_methods.py

```diff
@@ -221,8 +221,10 @@ def _var(a, axis=None, dtype=None, out=None, ddof=0, keepdims=False, *,
     if isinstance(arrmean, mu.ndarray):
         arrmean = um.true_divide(arrmean, div, out=arrmean, casting='unsafe',
                                  subok=False)
-    else:
+    elif hasattr(arrmean, "dtype"):
```

Comment:
This was the pattern we also use in the other places currently, right?

Reply:
Yup, currently it occurs about 4 times in `np.core._methods`.